### PR TITLE
Refactor style to match App

### PR DESCRIPTION
**Pull request by aspitz1**
This PR refactors the Movie to match the App and adds array of fetched movies.
* App starts with an empty array
* Fetch pushes the movie into array
* When the Movie is displayed the array is checked for the movie before the fetch
* If the array includes the movie the fetch is not made, instead the stored data is used

### CODE REVIEW

File: src/App/App.js [hunk 1]
```diff
@@ -40,30 +41,60 @@ class App extends Component {
       .then(data => this.setState({... this.state, movies: data.movies }))
   }
 
+  findMovie = (id) => {
```
Comment (aspitz1) [inline src/App/App.js:44]: Utility function to check for movie in `fetchedMovie` array

File: src/App/App.js [hunk 2]
```diff
@@ -40,30 +41,60 @@ class App extends Component {
...
-            releaseDate: data.movie['release_date'],
-            overview: data.movie.overview,
-            genres: data.movie.genres,
-            budget: data.movie.budget,
-            revenue: data.movie.revenue,
-            runtime: data.movie.runtime,
-            tagline: data.movie.tagline,
-            averageRating: data.movie['average_rating'],
+    const movie = this.findMovie(id);
+    if(movie) {
```
Comment (aspitz1) [inline src/App/App.js:50]: if move is stored, then the data from that is used to update the movie that should be displayed

Comment (mayakappen) [reply src/App/App.js:50]: this is going to reduce so many fetch requests nice job!!!

File: src/App/App.css [hunk 3]
```diff
@@ -2,7 +2,7 @@
   margin: 0;
   padding: 0;
   width: 100vw;
-  background-color: #000;
+  background-color: #445566;
```
Comment (mayakappen) [inline src/App/App.css:5]: oooh I'm excited to finally have the background color work right!

File: src/App/App.js [hunk 4]
```diff
@@ -40,30 +41,60 @@ class App extends Component {
...
+          genres: data.movie.genres,
+          budget: data.movie.budget,
+          revenue: data.movie.revenue,
+          runtime: data.movie.runtime,
+          tagline: data.movie.tagline,
+          averageRating: data.movie['average_rating'],
+          error: ''
+        }})
+      })
+      .catch(err =>{
```
Comment (mayakappen) [inline src/App/App.js:94]: ty for adding a catch as well!

File: src/Movie/Movie.js [hunk 5]
```diff
@@ -15,17 +15,19 @@ const Movie = ({ movieDetails }) => {
...
-                </ul>
+                <div className='movie-heading-description-container'>
+                    <h2>{title}</h2>
+                    <ul className="movie-heading-details">
+                        <li>Average Rating: {averageRating.toFixed(2)}</li>
+                        <li><span className='no-wrap'>{genres.join(' | ')}</span> <span className='no-wrap'>Released: {releaseDateFormatted}</span></li>
+                        <li><span className='no-wrap'>{runtime} Minutes</span></li>
+                        <li><span className='no-wrap'>Budget: ${budget.toLocaleString('en-US')}</span></li>
+                        <li><span className='no-wrap'>Revenue: ${revenue.toLocaleString('en-US')}</span></li>
+                    </ul>
```
Comment (mayakappen) [inline src/Movie/Movie.js:29]: elegant layout!

**Review by mayakappen — COMMENTED** [review]
Wonderful work! Less fetch requests is always good :)

**Pull request merged**